cogboard-webapp/src/components/widgets/dialogFields/SonarQubeMetricsInput.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import styled from '@emotion/styled/macro';
+
+import Input from '@material-ui/core/Input';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import ListItemText from '@material-ui/core/ListItemText';
+import Select from '@material-ui/core/Select';
+import Checkbox from '@material-ui/core/Checkbox';
+
+  const StyledFormControl = styled(FormControl)`
+      margin: theme.spacing(1),
+      minWidth: 120,
+      maxWidth: 300,
+  `;
+
+  const allMetrics = [
+    "blocker_violations",
+    "critical_violations",
+    "major_violations",
+    "minor_violations",
+    "info_violations",
+    "bugs",
+    "code_smells",
+    "vulnerabilities"
+  ];
+
+const SonarQubeMetricsInput = props => {
+  const { onChange } = props;
+  let { value } = props;
+  value = value || ['blocker_violations','critical_violations','major_violations','minor_violations'];
+
+  return (
+    <StyledFormControl>
+      <InputLabel htmlFor="select-multiple-checkbox">Metrics</InputLabel>
```

Comment:
It should take an id from dialogFields config through props. htmlFor and id fo the input.